pkgs/development/tools/verible/default.nix

```diff
@@ -13,18 +13,18 @@ let
 in
 buildBazelPackage rec {
   pname = "verible";
-  version = "0.0-2172-g238b6df6";
+  version = "0.0-2472-ga80124e1";
 
   # These environment variables are read in bazel/build-version.py to create
   # a build string. Otherwise it would attempt to extract it from .git/.
-  GIT_DATE = "2022-08-08";
+  GIT_DATE = "2022-10-21";
   GIT_VERSION = version;
 
   src = fetchFromGitHub {
     owner = "chipsalliance";
     repo = "verible";
     rev = "v${version}";
-    sha256 = "sha256-iOJhdbipuqqBiYGgk95d1c8bEK6Z16l16GuzYCQRc2g=";
+    sha256 = "sha256:0jpdxqhnawrl80pbc8544pyggdp5s3cbc7byc423d5v0sri2f96v";
```

Comment:
how'd you get this type of hash (asking because everything should give a hash with `-` instead of `:`)

Reply:
I got it from the ofborg log output (as I don't have an aarch64 linux to test at home).

Reply:
Oh, wait, this hash output: I think I extracted it from local `nix-build`: I usually just give it an empty hash, let nix-build complain about what _actual_ hash it wants and then copy it there.
There are probably better ways :)

Reply:
nix-build should now give a SRI hash too so i guess you're just on a old nix version

> There are probably better ways :)

that's what i do too and it's usually the best way, i've even got a keybind on kitty to copy it
`map ctrl+f4 kitten hints --program @ --type regex --regex '(?<=got.....)sha256.{45}'`

Reply:
Indeed, I had an old `nix-build` link in my nix-env.

Should I update the hash to the new form, or would you like the CI finish first ?

Reply:
update it when you update the package in a future pr